podman_compose.py

```diff
@@ -2352,7 +2341,12 @@ def _init_global_parser(parser):
         parser.add_argument("-v", "--version", help="show version", action="store_true")
         parser.add_argument(
             "--in-pod",
-            help="pod creation",
+            help=(
+                "Specify pod usage:\n"
+                "  'true' - create/use a pod named pod_<project name>\n"
+                "  'false'    - do not use a pod\n"
```

Comment:
Still weird formatting. Either don't add extra spaces or align start of the text on the same column, whichever is your preference.